npm specifiers in deno compile with @azure/storage-blob

Hello,

We have a facing issue with 'npm:@azure/storage-blob' this package. In Deno run command this package will work fine and there is no error but after i compile the related script there is an error below here;


```
error: Uncaught Error: Cannot find module '@azure/core-tracing'
Require stack:
- /tmp/deno-compile-ctapi/node_modules/localhost/@azure/core-http/3.0.2/dist/index.js
- /tmp/deno-compile-ctapi/node_modules/localhost/@azure/storage-blob/12.14.0/dist/index.js
- /tmp/deno-compile-ctapi/node_modules/localhost/@azure/storage-blob/12.14.0/dist/index.js
    at Function.Module._resolveFilename (ext:deno_node/01_require.js:840:15)
    at Function.Module._load (ext:deno_node/01_require.js:673:27)
    at Module.require (ext:deno_node/01_require.js:907:19)
    at require (ext:deno_node/01_require.js:1047:16)
    at Object.<anonymous> (file:///tmp/deno-compile-ctapi/node_modules/localhost/@azure/core-http/3.0.2/dist/index.js:20:19)
    at Object.<anonymous> (file:///tmp/deno-compile-ctapi/node_modules/localhost/@azure/core-http/3.0.2/dist/index.js:5467:4)
    at Module._compile (ext:deno_node/01_require.js:974:34)
    at Object.Module._extensions..js (ext:deno_node/01_require.js:1007:10)
    at Module.load (ext:deno_node/01_require.js:885:32)
    at Function.Module._load (ext:deno_node/01_require.js:719:12)
error: Uncaught Error: Cannot find module '@azure/core-tracing'
Require stack:
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -53,8 +53,8 @@ deno_bench_util = { version = "0.102.0", path = "./bench_util" }
 test_util = { path = "./test_util" }
 deno_lockfile = "0.14.1"
 deno_media_type = { version = "0.1.0", features = ["module_specifier"] }
-deno_npm = "0.8.0"
-deno_semver = "0.2.1"
+deno_npm = "0.8.1"
+deno_semver = "0.2.2"
 
 # exts
 deno_broadcast_channel = { version = "0.102.0", path = "./ext/broadcast_channel" }

